lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -164,7 +170,7 @@ def unstore(options = {})
 
         add_shopper_reference(post, options)
         add_merchant_account(post, options)
-        post[:recurringDetailReference] = options[:recurring_detail_reference]
+        add_recurring_detail_reference(post, options)
```

Comment:
This you have to leave it as it was before because this endpoint doesn't accept `selectedRecurringDetailReference` but it takes `recurringDetailReference` https://docs.adyen.com/api-explorer/Recurring/68/post/disable#request-recurringDetailReference

Reply:
Done!